src/lib/oauth2-service.ts

```diff
@@ -31,7 +31,7 @@ import { OAuth2Issuer } from './oauth2-issuer';
 import { assertIsString, assertIsValidTokenRequest } from './helpers';
 import type {
   JwtTransform,
-  MutableAuthorizeRedirectUri,
+  MutableRedirectUri,
```

Comment:
@poveden Note: From a pure type definition standpoint, this will be a breaking change, from the consumer standpoint.